### DISCUSSION TITLE
how to add custom riscv instruction on windows

### DISCUSSION THREAD

**Discussion by roman012285** [xpack-dev-tools/riscv-none-elf-gcc-xpack#24] [Q&A]
I am working with riscv-none-embedd-gcc-xpack I am trying to add a custome instruction to C code or to assembly code. There are few guides in google but they all refer to linux.
Please help

**Comment by ilg-ul** [comment]
Sorry, but this project just packs the GNU GCC sources, and does not customise the compiler.
